### ISSUE TITLE
adapted for tab capture

### ISSUE THREAD

**Issue by engelmav** [pulse0ne/eq8#34]
Hi,

Awesome extension. There are very few (findable) JS implementations of PEQ graphs.

I pulled your code and adapted it to do tab capture (rather than using an audio element).

Interested in a PR?

**Comment by pulse0ne** [comment]
Absolutely! I was actually planning on doing this (I’ve had some users complain that it doesn’t work on sites with <audio> elements not attached to the DOM), I’ve just not had time. Definitely interested if you have code to push!
